File: solutions/javascript/raindrops/1/raindrops.js
export const convert = (number) => {
    let word1 = "Pling"
    let word2 = "Plang"
    let word3 =  "Plong"
     
    if (number % 5 == 0 && number % 7 === 0 && number % 3 === 0) return word1 + word2 + word3
    if ( (number % 3 === 0 &&  number % 5 === 0)) return word1 + word2
    if (number % 3 === 0 &&  number % 7 === 0) return word1 + word3 
    if (number % 5 === 0 &&  number % 7 == 0)  return word2 + word3 
    if (number % 7 === 0 &&  (!number % 3 === 0 || !number & 7 === 0) ) return word3
    if (number % 5 === 0 ) return word2 
    if (number % 3 === 0) return  word1
    else return String(number)     
};

File: solutions/dart/raindrops/1/lib/raindrops.dart
class Raindrops {
   String convert (int n) {
      String answer = '';
      String wordKey = 'Pling';
      String wordKey2 = 'Plang';
      String wordKey3 = 'Plong';

      if ( n % 3 == 0) answer += wordKey;
      if ( n % 5 == 0) answer += wordKey2;
      if  ( n % 7 == 0) answer += wordKey3;
      if ( n % 3 != 0 && n % 5 != 0 && n % 7 != 0) answer = n.toString();
      return answer;
   }
}
